### PR TITLE
Compressed video recorder

### PR DESCRIPTION
**Pull request by 708yamaguchi**
I add option to record compressed image.

https://github.com/knorth55/app_manager_utils/issues/30

### CODE REVIEW

File: app_recorder/launch/video_recorder.launch [hunk 1]
```diff
@@ -3,10 +3,19 @@
   <arg name="video_path" />
   <arg name="video_title" />
   <arg name="video_topic_name" />
+  <arg name="use_compressed" default="false" />
+  <arg name="video_compressed_topic_name" />
```
Comment (knorth55) [inline app_recorder/launch/video_recorder.launch]:
```suggestion
  <arg name="video_compressed_topic_name" default="" />
```

Comment (knorth55) [reply app_recorder/launch/video_recorder.launch]: we need to set compressed topic name even when you don't want to use compressed if we don't have the default value.

**Review by knorth55 — CHANGES_REQUESTED** [review]

File: app_recorder/launch/video_recorder.launch (hunk 1 above)
Comment (708yamaguchi) [reply app_recorder/launch/video_recorder.launch]: Thank you for your advice.

I updated PR.

File: app_recorder/launch/audio_video_recorder.launch [hunk 2]
```diff
@@ -7,13 +7,22 @@
   <arg name="audio_sample_rate" />
   <arg name="audio_format" />
   <arg name="audio_sample_format" />
-  <arg name="video_topic_name" />
```
Comment (knorth55) [inline app_recorder/launch/audio_video_recorder.launch:10]: `video_topic_name` is required in any case.
do not change this.

**Review by knorth55 — CHANGES_REQUESTED** [review]

**Comment by 708yamaguchi** [comment]
I'm sorry, but could you teach me the difference between
```
<arg name="video_compressed_topic_name" default="" />
```
and
```
<arg name="video_topic_name" />
```
.

Why the former case needs default value and the latter case does not need default value?

**Comment by knorth55** [comment]
`video_topic_name` is required whenever `use_compressed` is `true` or `false`.
`video_compressed_topic_name` is only required when `use_compressed` is `true`.
thats why `video_topic_name` should not set default value and `video_topic_compressed_topic_name` should set default value.

**Comment by 708yamaguchi** [comment]
Thank you for your explanation.
I finally understand your idea.

I added the commit not to change tag of `video_topic_name`.

**Review by knorth55 — APPROVED** [review]

**Pull request merged**